### PR TITLE
[DX-705] Letzya plugin old structure

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

https://deploy-preview-3375--tyk-docs.netlify.app//docs/nightly/plugins

<br>

### Description

The plugin section after automatic navigation is completely flat and hard to follow. I put back the hierarchy we had before. 
We are working on improving the whole section but till it's ready, we need more structure. 

 

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/data/menu.yaml [hunk 1]
```diff
@@ -1700,162 +1700,187 @@ menu:
...
             show: True
+          - title: "Serving plugins"
+            category: Directory
+            show: True
+            menu:
+            - title: "Serving Plugins to Tyk Gateway"
+              path: /plugins/how-to-serve-plugins
+              category: Page
+              show: True
+            - title: "Plugin server"
```
Comment (dcs3spp) [inline tyk-docs/data/menu.yaml]: Should we use this for the menu title?

```suggestion
            - title: "Packaging plugins"
```

File: tyk-docs/data/menu.yaml [hunk 2]
```diff
@@ -1700,162 +1700,187 @@ menu:
...
+              show: True
+            - title: "Rich Plugins - Data Structures"
+              path: /plugins/supported-languages/rich-plugins/rich-plugins-data-structures
+              category: Page
+              show: True
+            - title: "Rich Plugin - Python"
+              category: Directory
+              show: True
+              menu: 
+              - title: "Custom Authentication With a Python Plugin"
```
Comment (dcs3spp) [inline tyk-docs/data/menu.yaml]: Should we reorder the Python menu items to:

- Python or Overview?
- Tutorial Add a Python plugin bundle to your Gateway
- Tutorial Custom Authentication
- Tyk Python API methods
- Python Performance

Comment (dcs3spp) [inline tyk-docs/data/menu.yaml]:
```suggestion
              - title: "Tutorial Custom Authentication"
```

**Review by dcs3spp — COMMENTED** [review]
Many thanks Yaara for adding structure. I have made minor suggestions concerning the ordering of menu items in the Python Plugins section and the naming of the plugins bundling menu item

File: tyk-docs/data/menu.yaml (hunk 1 above)
Comment (letzya) [reply tyk-docs/data/menu.yaml]: I think bundling is a bit of an internal term, so I wanted to use a generic phrase that is clearer to the reader

File: tyk-docs/data/menu.yaml (hunk 2 above)
Comment (letzya) [reply tyk-docs/data/menu.yaml]: are you sure it is read well?

Comment (letzya) [reply tyk-docs/data/menu.yaml]: I reordered it, please check

File: tyk-docs/data/menu.yaml (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/data/menu.yaml]: 🤔 My thinking was that plugin server might imply the server from which the plugin bundles are fetched. How about Packaging plugins?

**Review by dcs3spp — COMMENTED** [review]
lgtm, 
🤔 For the Plugin server menu link title, my thinking was that plugin server might imply the server from which the plugin bundles are fetched. How about having the link title as Packaging plugins?

**Review by dcs3spp — APPROVED** [review]
lgtm
